docs/vercel.json

```diff
@@ -58,11 +62,6 @@
       "has": [{ "type": "host", "value": "qwik.authjs.dev" }],
       "destination": "https://authjs.dev/reference/qwik"
     },
-    {
-      "source": "/:path(.*)",
-      "has": [{ "type": "host", "value": "cli.authjs.dev" }],
-      "destination": "https://github.com/nextauthjs/cli"
```

Comment:
Why are we dropping this redirect? 🤔

Reply:
I'm sorry. I included unintended changes.
It has been corrected to the correct one.